CHANGES.txt

```diff
@@ -15,6 +15,8 @@ the appropriate section of the docs.
 Breaking Changes
 ================
 
+ - The ``license.enterprise`` setting is set to ``true`` by default.
```

Comment:
i would also mention what's breaking here, e.g: "If you start crate enterprise without a valid enterprise license you are entering the unlicensed mode"